.github/workflows/package.yml

```diff
@@ -0,0 +1,30 @@
+name: Package
+on:
+  push: {}
```

Comment:
i assume we want to run this all the time anyway to make sure packaging works, even if we dont actually "release" it?

Reply:
Yah, that was my idea. If we only have it `on.push.tags` but it was broken in the intermediate time, we won't know until we push the tag that it's broken.

Do you think that's an OK way to approach it? Or is there a better way?